### DISCUSSION TITLE
Items will fall to the floor when two players put things into the same container at same time

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14853] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

if two players put items into the same container (steel cabinet, deconstructor...) at same time
only one of player can successfully put items. Items put by another player will fall to the floor

### Reproduction steps

1. start a multiplayer game
2. make two players put items into same cabinet/deconstructor
3. items fall to the floor

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Crazyjr20** [comment]
Age old bug

**Comment by cl0ck-byte** [comment]
Duplicate of #14855 but I'd leave it in since it's two issues combined into one on the mentioned issue, worse for tracking
